packages/block-library/src/group/editor.scss

```diff
@@ -3,9 +3,6 @@
  */
 
 .wp-block-group {
-	// Zero out the baseline margin that is set for every block in the editor.
-	margin-top: 0;
-	margin-bottom: 0;
```

Comment:
I think this is not needed with my change here https://github.com/WordPress/gutenberg/pull/24966/files#r481034215